bluetooth/api/blip_api.py

```diff
@@ -267,22 +307,23 @@ def main(dbsetting: 'path/to/config.cfg' = None,
 
     dbset, api_settings = load_config(dbsetting)
 
-    # Access the API using zeep
+    # Access the API using zeep (Why do we need to sleep the connection when that function exist)
     LOGGER.info('Fetching config from blip server')
     blip, config = get_wsdl_client(api_settings['WSDLfile'], direct, live)
 
     try:
-        db = DB(**dbset)
+        conn = _get_db(dbset)
     except InternalError as _:
         LOGGER.error('Connection error to RDS, sleeping for ten minutes')
         sleep(60 * 10)
-        db = DB(**dbset)
+        conn = _get_db(dbset)
```

Comment:
I think this part can be changed to similar to the other _get_db() block

Reply:
like which ones ? 😵

Reply:
dis
https://github.com/CityofToronto/bdit_data-sources/blob/2e9a12801bfa6382f3c8113824a469961361afa2/bluetooth/api/blip_api.py#L122-L127

Reply:
you have to link to the permalink

![image](https://user-images.githubusercontent.com/10623872/178346646-f531cadd-c77b-4738-b474-67b5120fa1ee.png)